README.md

```diff
@@ -13,3 +13,159 @@
 
 GraphQL Service runs on port 23431 at `/graphql` by default
 
+
+## Queries
+Here are some of the important GraphQL queries:
+
+### 1. Get all traces in provided time range
+
+```graphql
+curl -s localhost:2020/graphql -H 'Content-Type: application/graphql' -d\
```

Comment:
The above statement says "GraphQL Service runs on port 23431 at `/graphql` by default", but we are trying to hit `localhost:2020`

Reply:
The above statement was there earlier! You want to change that?

Reply:
Yeah. As long as the things are consistent and correct